### ISSUE TITLE
jfltexx:failed to write /dev/block/mmcblk0p9: I/O error

### ISSUE THREAD

**Issue by dkati** [TeamWin/Team-Win-Recovery-Project#874]
in jfltexx we have to use /fota for /misc , not the /pad

[recovery.txt](https://github.com/TeamWin/Team-Win-Recovery-Project/files/836220/recovery.txt)

**Comment by hja-de** [comment]
Same error appears on my device using twrp-3.1.0.0-jfltexx

**Comment by howels** [comment]
Same error on my unit. "failed to write /dev/block/mmcblock0p9: I/O error" when flashing on twrp-3.1.0.0-jtflexx.

**Comment by fm99** [comment]
I have this problem too - now I'm back to 3.0.2.0 ...

**Comment by dkati** [comment]
there is no reason to worry about...its just a message.doesnt affect the operation guys

**Comment by howels** [comment]
Operation is affected, in that the recovery script ought to flash a new image but instead it just sits and does nothing with that warning message. Unless lineageOS isn't supplying a recovery script? Would be odd but not impossible.

**Comment by walttheboss** [comment]
I agree.  The zip does not flash in 3.1.  Flashing a rom on i9508(essentially a i9505) failed with 3.1 and worked on 3.0.2.  Same error message noted.
